graphql/types/Order.graphql

```diff
@@ -92,6 +92,8 @@ type OrderItemPaymentConnectorResponse {
   tid: String
   returnCode: String
   message: String
+  meoWalletReference: String
```

Comment:
I would rather move those fields to a generic field, like:

```graphql
scalar ConnectorAdditionalData

type OrderItemPaymentConnectorResponse {
  tid: String
  returnCode: String
  message: String
  additionalData: ConnectorAdditionalData
}
```

Reply:
This doesn't work, a scalar type can't have subFields.

Reply:
It worked with scalar type, it is returning sub-fields.